example/example-Tzip16Https.ts

```diff
@@ -0,0 +1,85 @@
+import { MichelsonMap, TezosToolkit } from '@taquito/taquito';
+import { importKey } from '@taquito/signer';
+import { b58cencode, char2Bytes, Prefix, prefix } from '@taquito/utils';
+import { tacoContractTzip16 } from "../integration-tests/data/modified-taco-contract"
+
+const provider = 'https://api.tez.ie/rpc/granadanet';
+
+const nodeCrypto = require('crypto');
+
+async function createAddress() {
```

Comment:
the function is not used and could be removed

Reply:
removed.